src/Parser/Parser.hs

```diff
@@ -176,18 +188,13 @@ notAlreadyInUse x = do
     then unexpected $ "redefinition of " ++ parsed
     else return parsed
 
--- | Valid character set that can be used to describe a game (after the first lead character)
-gameIdentifierChars :: [Char]
-gameIdentifierChars = ['a'..'z']++['A'..'Z']++['0'..'9']++"_"
-
--- | Identifies a valid game name, also used for types
--- nearly identical to 'capIdentifier', just includes underscores
+-- | Identifies a valid game name
 gameIdentifier :: ParsecT String u Identity [Char]
-gameIdentifier = lexeme ((:) <$> upper <*> (many (oneOf gameIdentifierChars)))
+gameIdentifier = capIdentifier
```

Comment:
We had an issue before with underscores related to this, with identifier, but there are tests against that. Since it passed, the expected behavior looks to be preserved properly